src/Microsoft.TemplateEngine.Cli/TemplatePackageCoordinator.cs

```diff
@@ -86,7 +86,7 @@ internal void DisplayUpdateCheckResult(CheckUpdateResult versionCheckResult, ICo
             }
             else
             {
-                HandleUpdateCheckErrors(versionCheckResult);
+                HandleUpdateCheckErrors(versionCheckResult, ignoreLocalPackageNotFound: true);
```

Comment:
minor: specifying the 2nd parameter `ignoreLocalPackageNotFound` is not necessary.

`HandleUpdateCheckErrors(versionCheckResult, true);` would be clear enough in this case.

Reply:
Please consider this comment for all such cases.

Reply:
@donJoseLuis we prefer named bool parameters for code readability.
In case `HandleUpdateCheckErrors(versionCheckResult, true);` it is not clear what `true` does.

Reply:
@donJoseLuis I added this based on a suggestion to explicitly name the parameter: https://github.com/dotnet/templating/pull/4502/commits/a709c9b40928aa6b9a435a8c0217d9b65e0371e4#r834073047
While not strongly opinionated I actually feel that explicit name adds clarity